components/kyma-environment-broker/internal/gardener/client.go

```diff
@@ -38,6 +38,18 @@ func NewGardenerSecretsInterface(gardenerClusterCfg *restclient.Config, gardener
 	return gardenerClusterClient.CoreV1().Secrets(gardenerNamespace), nil
 }
 
+func NewGardenerShootInterface(gardenerClusterCfg *restclient.Config, gardenerProjectName string) (gardener_apis.ShootInterface, error) {
+
+	gardenerNamespace := fmt.Sprintf("garden-%s", gardenerProjectName)
```

Comment:
minor: there are two places which creates namespace using the pattern, please extract it to a separate function